packages/skeleton-svelte/src/lib/internal/test-utils.ts

```diff
@@ -0,0 +1,11 @@
+import { createRawSnippet } from 'svelte';
+import type { Snippet } from 'svelte';
+
+/** Use `mockSnippet(value)` to create `<span>{value}</span>` snippet content. */
+export function mockSnippet(key: string): Snippet<[]> {
```

Comment:
No critique, just wanted to say this is really cool!

Reply:
It really is!